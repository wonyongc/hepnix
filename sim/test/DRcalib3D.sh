#!/bin/bash

cd /afs/cern.ch/user/w/wochung/private/dual-readout/install/test
source DRenv.sh
/afs/cern.ch/user/w/wochung/private/k4FWCore/install/bin/k4run runDRcalib3D.py
