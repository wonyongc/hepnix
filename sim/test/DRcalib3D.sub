executable            = DRcalib3D.sh
output                = output/DRcalib3D$(ClusterId).$(ProcId).out
error                 = output/DRcalib3D$(ClusterId).$(ProcId).err
log                   = output/DRcalib3D$(ClusterId).$(ProcId).log
should_transfer_files = Yes
transfer_input_files = digi.root, calib.csv, velo.root
when_to_transfer_output = ON_EXIT
+MaxRuntime = 3600
queue

