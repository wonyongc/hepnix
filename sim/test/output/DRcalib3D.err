DRenv.sh: line 20: export: `-w': not a valid identifier
DRenv.sh: line 29: export: `-ne': not a valid identifier
DRenv.sh: line 29: export: `\033]0;/afs/cern.ch/user/w/wochung/private/dual-readout/install/test\007': not a valid identifier
DRenv.sh: line 74: export: `-lz': not a valid identifier
DRenv.sh: line 74: export: `-lGL': not a valid identifier
DRenv.sh: line 74: export: `-lGLU': not a valid identifier
DRenv.sh: line 74: export: `-lfontconfig': not a valid identifier
DRenv.sh: line 74: export: `-lfreetype': not a valid identifier
DRenv.sh: line 74: export: `-lX11': not a valid identifier
DRenv.sh: line 74: export: `-lncurses': not a valid identifier
DRenv.sh: line 74: export: `-lpcre': not a valid identifier
DRenv.sh: line 74: export: `-ldl': not a valid identifier
DRenv.sh: line 74: export: `-lgfortran': not a valid identifier
DRenv.sh: line 74: export: `-lm': not a valid identifier
DRenv.sh: line 74: export: `-lquadmath': not a valid identifier
DRenv.sh: line 74: export: `-lutil': not a valid identifier
DRenv.sh: line 74: export: `-lm': not a valid identifier
DRenv.sh: line 74: export: `-L/cvmfs/sft.cern.ch/lcg/views/LCG_100/x86_64-centos7-gcc8-opt/lib': not a valid identifier
DRenv.sh: line 75: export: `-lz': not a valid identifier
DRenv.sh: line 75: export: `-lGL': not a valid identifier
DRenv.sh: line 75: export: `-lGLU': not a valid identifier
DRenv.sh: line 75: export: `-lfontconfig': not a valid identifier
DRenv.sh: line 75: export: `-lfreetype': not a valid identifier
DRenv.sh: line 75: export: `-lX11': not a valid identifier
DRenv.sh: line 75: export: `-lncurses': not a valid identifier
DRenv.sh: line 75: export: `-lpcre': not a valid identifier
DRenv.sh: line 75: export: `-ldl': not a valid identifier
DRenv.sh: line 75: export: `-lgfortran': not a valid identifier
DRenv.sh: line 75: export: `-lm': not a valid identifier
DRenv.sh: line 75: export: `-lquadmath': not a valid identifier
DRenv.sh: line 75: export: `-lutil': not a valid identifier
DRenv.sh: line 75: export: `-lm': not a valid identifier
DRenv.sh: line 75: export: `-L/cvmfs/sft.cern.ch/lcg/views/LCG_100/x86_64-centos7-gcc8-opt/lib': not a valid identifier
Warning in <TInterpreter::ReadRootmapFile>: class  edm4hep::ObjectID found in libedm4drDict.so  is already in libedm4hepDict.so 
Warning in <TClassTable::Add>: class edm4hep::ObjectID already in TClassTable
Warning in <TGeoManager>: Changing system of units to ROOT units (cm, s, GeV).
