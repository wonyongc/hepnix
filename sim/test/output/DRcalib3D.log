000 (1369727.000.000) 04/18 18:16:45 Job submitted from host: <188.185.125.71:9618?addrs=188.185.125.71-9618+[2001-1458-d00-4a--100-15e]-9618&alias=bigbird28.cern.ch&noUDP&sock=schedd_1933_74a8>
...
040 (1369727.000.000) 04/18 18:19:52 Started transferring input files
	Transferring to host: <128.142.51.200:9618?addrs=128.142.51.200-9618+[2001-1458-301-3--100-5]-9618&alias=b7g02p5640.cern.ch&noUDP&sock=slot1_39_4620_f535_13445>
...
040 (1369727.000.000) 04/18 18:19:53 Finished transferring input files
...
001 (1369727.000.000) 04/18 18:19:54 Job executing on host: <128.142.51.200:9618?addrs=128.142.51.200-9618+[2001-1458-301-3--100-5]-9618&alias=b7g02p5640.cern.ch&noUDP&sock=startd_3179_1ba9>
...
006 (1369727.000.000) 04/18 18:20:03 Image size of job updated: 305368
	14  -  MemoryUsage of job (MB)
	13536  -  ResidentSetSize of job (KB)
...
006 (1369727.000.000) 04/18 18:25:04 Image size of job updated: 6769780
	5977  -  MemoryUsage of job (MB)
	6120024  -  ResidentSetSize of job (KB)
...
006 (1369727.000.000) 04/18 18:30:04 Image size of job updated: 18979912
	15225  -  MemoryUsage of job (MB)
	15589976  -  ResidentSetSize of job (KB)
...
006 (1369727.000.000) 04/18 18:35:05 Image size of job updated: 21035680
	17116  -  MemoryUsage of job (MB)
	17526340  -  ResidentSetSize of job (KB)
...
006 (1369727.000.000) 04/18 18:38:29 Image size of job updated: 25864280
	17116  -  MemoryUsage of job (MB)
	17526376  -  ResidentSetSize of job (KB)
...
040 (1369727.000.000) 04/18 18:38:29 Started transferring output files
...
040 (1369727.000.000) 04/18 18:38:30 Finished transferring output files
...
005 (1369727.000.000) 04/18 18:38:30 Job terminated.
	(1) Normal termination (return value 139)
		Usr 0 00:12:50, Sys 0 00:00:59  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:12:50, Sys 0 00:00:59  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	36816  -  Run Bytes Sent By Job
	9970648  -  Run Bytes Received By Job
	36816  -  Total Bytes Sent By Job
	9970648  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.22        1         1 
	   Disk (KB)            :  9794       10000   1415932 
	   Memory (MB)          : 17116        2000      2000 

	Job terminated of its own accord at 2023-04-18T16:38:29Z.
...
000 (1369731.000.000) 04/18 18:42:16 Job submitted from host: <188.185.125.71:9618?addrs=188.185.125.71-9618+[2001-1458-d00-4a--100-15e]-9618&alias=bigbird28.cern.ch&noUDP&sock=schedd_1933_74a8>
...
040 (1369731.000.000) 04/18 18:44:43 Started transferring input files
	Transferring to host: <128.142.193.173:9618?addrs=128.142.193.173-9618+[2001-1458-301-8--100-29]-9618&alias=b7g21p2260.cern.ch&noUDP&sock=slot1_33_3293_f75f_8232>
...
040 (1369731.000.000) 04/18 18:44:43 Finished transferring input files
...
001 (1369731.000.000) 04/18 18:44:44 Job executing on host: <128.142.193.173:9618?addrs=128.142.193.173-9618+[2001-1458-301-8--100-29]-9618&alias=b7g21p2260.cern.ch&noUDP&sock=startd_2834_777d>
...
006 (1369731.000.000) 04/18 18:44:53 Image size of job updated: 143632
	11  -  MemoryUsage of job (MB)
	10788  -  ResidentSetSize of job (KB)
...
006 (1369731.000.000) 04/18 18:49:53 Image size of job updated: 6787600
	5617  -  MemoryUsage of job (MB)
	5750820  -  ResidentSetSize of job (KB)
...
006 (1369731.000.000) 04/18 18:54:53 Image size of job updated: 18977008
	17980  -  MemoryUsage of job (MB)
	18411500  -  ResidentSetSize of job (KB)
...
006 (1369731.000.000) 04/18 18:57:22 Image size of job updated: 21062132
	17981  -  MemoryUsage of job (MB)
	18411668  -  ResidentSetSize of job (KB)
...
040 (1369731.000.000) 04/18 18:57:22 Started transferring output files
...
040 (1369731.000.000) 04/18 18:57:22 Finished transferring output files
...
005 (1369731.000.000) 04/18 18:57:22 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:11:37, Sys 0 00:00:28  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:11:37, Sys 0 00:00:28  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	20468  -  Run Bytes Sent By Job
	9970648  -  Run Bytes Received By Job
	20468  -  Total Bytes Sent By Job
	9970648  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :  9777       10000   1338501 
	   Memory (MB)          : 17981        2000      2000 

	Job terminated of its own accord at 2023-04-18T16:57:22Z.
...
