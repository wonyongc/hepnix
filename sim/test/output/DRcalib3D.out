ApplicationMgr    SUCCESS 
====================================================================================================================================
                                                   Welcome to ApplicationMgr (GaudiCoreSvc v35r1)
                                          running on b7g21p2260.cern.ch on Tue Apr 18 18:44:58 2023
====================================================================================================================================
ApplicationMgr       INFO Application Manager Configured successfully
PersistencyIO    INFO  +++ Set Streamer to dd4hep::OpaqueDataBlock
GeoSvc               INFO loading geometry from file:  'file:/afs/cern.ch/user/w/wochung/private/dual-readout/install/share/compact/DRcalo.xml'
DD4hep           WARN  ++ STD conditions NOT defined by client. NTP defaults taken.
Detector         INFO  *********** Created World volume with size: 1000 1000 1000
Compact          INFO  ++ Converted subdetector:DRcalo of type ddDRcalo [DRcaloSiPMSD]
Detector         INFO  +++ Patching names of anonymous shapes....
DD4hepVolumeManager INFO  **** running plugin DD4hepVolumeManager ! 
VolumeManager    INFO   - populating volume ids - be patient ...
VolumeManager    INFO   - populating volume ids - done. 52072 nodes.
VolumeManager    INFO  +++ Volume manager populated and loaded.
GeoSvc               INFO DD4Hep geometry SUCCESSFULLY built
GeoSvc               INFO Geant4 geometry SUCCESSFULLY built
DRcalib2D           DEBUG Property update for OutputLevel : new value = 2
DRcalib2D           DEBUG Initialize base class GaudiCommon<Algorithm>
DRcalib2D           DEBUG could not locate CounterSummarySvc, no counter summary will be made
DRcalib2D           DEBUG List of ALL properties of DRcalib2D/DRcalib2D  #properties = 38
DRcalib2D           DEBUG Property ['Name': Value] =  'GeoSvc':GeoSvc
DRcalib2D           DEBUG Property ['Name': Value] =  'sampling':0.10000000
DRcalib2D           DEBUG Property ['Name': Value] =  'readoutName':'DRcaloSiPMreadout'
DRcalib2D           DEBUG Property ['Name': Value] =  'calibPath':'calib.csv'
DRcalib2D           DEBUG Property ['Name': Value] =  'RequireObjects':[  ]
DRcalib2D           DEBUG Property ['Name': Value] =  'VetoObjects':[  ]
DRcalib2D           DEBUG Property ['Name': Value] =  'StatEntityList':[  ]
DRcalib2D           DEBUG Property ['Name': Value] =  'CounterList':[ '.*' ]
DRcalib2D           DEBUG Property ['Name': Value] =  'UseEfficiencyRowFormat':True
DRcalib2D           DEBUG Property ['Name': Value] =  'EfficiencyRowFormat':' |*%|-48.48s|%|50t||%|10d| |%|11.5g| |(%|#9.6g| +- %|-#9.6g|)%%|   -------   |   -------   |'
DRcalib2D           DEBUG Property ['Name': Value] =  'RegularRowFormat':' | %|-48.48s|%|50t||%|10d| |%|11.7g| |%|#11.5g| |%|#11.5g| |%|#12.5g| |%|#12.5g| |'
DRcalib2D           DEBUG Property ['Name': Value] =  'StatTableHeader':' |    Counter                                      |     #     |    sum     | mean/eff^* | rms/err^*  |     min     |     max     |'
DRcalib2D           DEBUG Property ['Name': Value] =  'Context':''
DRcalib2D           DEBUG Property ['Name': Value] =  'TypePrint':True
DRcalib2D           DEBUG Property ['Name': Value] =  'PrintEmptyCounters':False
DRcalib2D           DEBUG Property ['Name': Value] =  'PropertiesPrint':False
DRcalib2D           DEBUG Property ['Name': Value] =  'ErrorsPrint':True
DRcalib2D           DEBUG Property ['Name': Value] =  'RootInTES':''
DRcalib2D           DEBUG Property ['Name': Value] =  'FilterCircularDependencies':True
DRcalib2D           DEBUG Property ['Name': Value] =  'Blocking':False
DRcalib2D           DEBUG Property ['Name': Value] =  'NeededResources':[  ]
DRcalib2D           DEBUG Property ['Name': Value] =  'Cardinality':1
DRcalib2D           DEBUG Property ['Name': Value] =  'RegisterForContextService':True
DRcalib2D           DEBUG Property ['Name': Value] =  'MonitorService':'MonitorSvc'
DRcalib2D           DEBUG Property ['Name': Value] =  'Timeline':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditStop':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditStart':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditFinalize':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditExecute':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditRestart':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditReinitialize':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditInitialize':False
DRcalib2D           DEBUG Property ['Name': Value] =  'AuditAlgorithms':False
DRcalib2D           DEBUG Property ['Name': Value] =  'ErrorMax':1
DRcalib2D           DEBUG Property ['Name': Value] =  'Enable':True
DRcalib2D           DEBUG Property ['Name': Value] =  'OutputLevel':2
DRcalib2D           DEBUG Property ['Name': Value] =  'ExtraOutputs':[]
DRcalib2D           DEBUG Property ['Name': Value] =  'ExtraInputs':[]
DRcalib2D            INFO DRcalib2D initialized
DRcalib2D           DEBUG input handles: 1
DRcalib2D           DEBUG output handles: 1
DRcalib2D           DEBUG Changing DigiCalorimeterHits to /Event/DigiCalorimeterHits
DRcalib2D           DEBUG Changing DRcalo2dHits to /Event/DRcalo2dHits
DRcalib2D           DEBUG Data Deps for DRcalib2D
  + INPUT  '/Event/DigiCalorimeterHits'
  + OUTPUT '/Event/DRcalo2dHits'
DRcalib3D           DEBUG Property update for OutputLevel : new value = 2
DRcalib3D           DEBUG Initialize base class GaudiCommon<Algorithm>
DRcalib3D           DEBUG could not locate CounterSummarySvc, no counter summary will be made
DRcalib3D           DEBUG List of ALL properties of DRcalib3D/DRcalib3D  #properties = 46
DRcalib3D           DEBUG Property ['Name': Value] =  'GeoSvc':GeoSvc
DRcalib3D           DEBUG Property ['Name': Value] =  'cherenScale':1.1250000
DRcalib3D           DEBUG Property ['Name': Value] =  'scintScale':1.2320000
DRcalib3D           DEBUG Property ['Name': Value] =  'nbins':900
DRcalib3D           DEBUG Property ['Name': Value] =  'threshold':0.010000000
DRcalib3D           DEBUG Property ['Name': Value] =  'gateLength':90.000000
DRcalib3D           DEBUG Property ['Name': Value] =  'gateStart':10.000000
DRcalib3D           DEBUG Property ['Name': Value] =  'sampling':0.10000000
DRcalib3D           DEBUG Property ['Name': Value] =  'scintProf':'VeloS_pfx'
DRcalib3D           DEBUG Property ['Name': Value] =  'cherenProf':'VeloC_pfx'
DRcalib3D           DEBUG Property ['Name': Value] =  'veloFile':'velo.root'
DRcalib3D           DEBUG Property ['Name': Value] =  'readoutName':'DRcaloSiPMreadout'
DRcalib3D           DEBUG Property ['Name': Value] =  'RequireObjects':[  ]
DRcalib3D           DEBUG Property ['Name': Value] =  'VetoObjects':[  ]
DRcalib3D           DEBUG Property ['Name': Value] =  'StatEntityList':[  ]
DRcalib3D           DEBUG Property ['Name': Value] =  'CounterList':[ '.*' ]
DRcalib3D           DEBUG Property ['Name': Value] =  'UseEfficiencyRowFormat':True
DRcalib3D           DEBUG Property ['Name': Value] =  'EfficiencyRowFormat':' |*%|-48.48s|%|50t||%|10d| |%|11.5g| |(%|#9.6g| +- %|-#9.6g|)%%|   -------   |   -------   |'
DRcalib3D           DEBUG Property ['Name': Value] =  'RegularRowFormat':' | %|-48.48s|%|50t||%|10d| |%|11.7g| |%|#11.5g| |%|#11.5g| |%|#12.5g| |%|#12.5g| |'
DRcalib3D           DEBUG Property ['Name': Value] =  'StatTableHeader':' |    Counter                                      |     #     |    sum     | mean/eff^* | rms/err^*  |     min     |     max     |'
DRcalib3D           DEBUG Property ['Name': Value] =  'Context':''
DRcalib3D           DEBUG Property ['Name': Value] =  'TypePrint':True
DRcalib3D           DEBUG Property ['Name': Value] =  'PrintEmptyCounters':False
DRcalib3D           DEBUG Property ['Name': Value] =  'PropertiesPrint':False
DRcalib3D           DEBUG Property ['Name': Value] =  'ErrorsPrint':True
DRcalib3D           DEBUG Property ['Name': Value] =  'RootInTES':''
DRcalib3D           DEBUG Property ['Name': Value] =  'FilterCircularDependencies':True
DRcalib3D           DEBUG Property ['Name': Value] =  'Blocking':False
DRcalib3D           DEBUG Property ['Name': Value] =  'NeededResources':[  ]
DRcalib3D           DEBUG Property ['Name': Value] =  'Cardinality':1
DRcalib3D           DEBUG Property ['Name': Value] =  'RegisterForContextService':True
DRcalib3D           DEBUG Property ['Name': Value] =  'MonitorService':'MonitorSvc'
DRcalib3D           DEBUG Property ['Name': Value] =  'Timeline':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditStop':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditStart':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditFinalize':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditExecute':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditRestart':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditReinitialize':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditInitialize':False
DRcalib3D           DEBUG Property ['Name': Value] =  'AuditAlgorithms':False
DRcalib3D           DEBUG Property ['Name': Value] =  'ErrorMax':1
DRcalib3D           DEBUG Property ['Name': Value] =  'Enable':True
DRcalib3D           DEBUG Property ['Name': Value] =  'OutputLevel':2
DRcalib3D           DEBUG Property ['Name': Value] =  'ExtraOutputs':[]
DRcalib3D           DEBUG Property ['Name': Value] =  'ExtraInputs':[]
DRcalib3D            INFO DRcalib3D initialized
DRcalib3D           DEBUG input handles: 3
DRcalib3D           DEBUG output handles: 2
DRcalib3D           DEBUG Changing DRcalo2dHits to /Event/DRcalo2dHits
DRcalib3D           DEBUG Changing DigiCalorimeterHits to /Event/DigiCalorimeterHits
DRcalib3D           DEBUG Changing DigiWaveforms to /Event/DigiWaveforms
DRcalib3D           DEBUG Changing DRpostprocTime to /Event/DRpostprocTime
DRcalib3D           DEBUG Changing DRcalo3dHits to /Event/DRcalo3dHits
DRcalib3D           DEBUG Data Deps for DRcalib3D
  + INPUT  '/Event/DRcalo2dHits'
  + INPUT  '/Event/DigiCalorimeterHits'
  + INPUT  '/Event/DigiWaveforms'
  + OUTPUT '/Event/DRcalo3dHits'
  + OUTPUT '/Event/DRpostprocTime'
EventLoopMgr      WARNING Unable to locate service "EventSelector" 
EventLoopMgr      WARNING No events will be processed from external input.
ApplicationMgr       INFO Application Manager Initialized successfully
ApplicationMgr       INFO Application Manager Started successfully
vector<float>
vector<float>
vector<float>
vector<float>
vector<float>
vector<float>
vector<float>
vector<float>
ApplicationMgr       INFO Application Manager Stopped successfully
DRcalib2D           DEBUG Finalize base class GaudiAlgorithm
DRcalib2D           DEBUG Tools to release :
DRcalib2D           DEBUG Services to release :
DRcalib3D           DEBUG Finalize base class GaudiAlgorithm
DRcalib3D           DEBUG Tools to release :
DRcalib3D           DEBUG Services to release :
EventLoopMgr         INFO Histograms converted successfully according to request.
ApplicationMgr       INFO Application Manager Finalized successfully
ApplicationMgr       INFO Application Manager Terminated successfully
 -->  PodioInput -->  DRcalib2D -->  DRcalib3D -->  PodioOutput 

Option name: PodioInput.OutputLevel PodioInput.OutputLevel 0
Option name: PodioInput.collections PodioInput.collections ['DigiCalorimeterHits', 'DigiWaveforms', 'RawTimeStructs', 'RawCalorimeterHits', 'SimCalorimeterHits', 'Sim3dCalorimeterHits', 'RawWavlenStructs', 'GenParticles', 'Leakages']
Option name: DRcalib3D.OutputLevel DRcalib3D.OutputLevel 2
Option name: DRcalib3D.readoutName DRcalib3D.readoutName DRcaloSiPMreadout
Option name: DRcalib3D.veloFile DRcalib3D.veloFile velo.root
Option name: DRcalib3D.cherenProf DRcalib3D.cherenProf VeloC_pfx
Option name: DRcalib3D.scintProf DRcalib3D.scintProf VeloS_pfx
Option name: DRcalib3D.sampling DRcalib3D.sampling 0.1
Option name: DRcalib3D.gateStart DRcalib3D.gateStart 10.0
Option name: DRcalib3D.gateLength DRcalib3D.gateLength 90.0
Option name: DRcalib3D.threshold DRcalib3D.threshold 0.01
Option name: DRcalib3D.nbins DRcalib3D.nbins 900
Option name: DRcalib3D.scintScale DRcalib3D.scintScale 1.232
Option name: DRcalib3D.cherenScale DRcalib3D.cherenScale 1.125
Option name: PodioOutput.OutputLevel PodioOutput.OutputLevel 0
Option name: PodioOutput.filename PodioOutput.filename reco3D.root
Option name: PodioOutput.outputCommands PodioOutput.outputCommands ['keep *']
Option name: PodioOutput.remoteFilename PodioOutput.remoteFilename 
Option name: GeoSvc.OutputLevel GeoSvc.OutputLevel 0
Option name: GeoSvc.detectors GeoSvc.detectors ['file:/afs/cern.ch/user/w/wochung/private/dual-readout/install/share/compact/DRcalo.xml']
Option name: EventDataSvc.OutputLevel EventDataSvc.OutputLevel 0
Option name: EventDataSvc.input EventDataSvc.input digi.root
Option name: EventDataSvc.FirstEventEntry EventDataSvc.FirstEventEntry 0
Option name: DRcalib2D.OutputLevel DRcalib2D.OutputLevel 2
Option name: DRcalib2D.calibPath DRcalib2D.calibPath calib.csv
Option name: DRcalib2D.readoutName DRcalib2D.readoutName DRcaloSiPMreadout
Option name: DRcalib2D.sampling DRcalib2D.sampling 0.1
